kolibri/core/assets/src/kolibri_core_app.js

```diff
@@ -1,8 +1,32 @@
 'use strict';
 
-var Kolibri = require('./mediator/mediator');
-var log = require('loglevel');
+var Mediator = require('./mediator/mediator');
+var logging = require('loglevel');
+var _ = require('lodash');
 
-log.setDefaultLevel(2);
+var public_methods = [
+    'register_plugin_async',
+    'register_plugin_sync',
+    'stop_listening',
+    'trigger'
+];
 
-log.info('Kolibri Initialized');
+// Bind 'this' value for public methods - those that will be exposed in the Facade.
+_.bindAll(Mediator, public_methods);
+
+var lib = {
+    Backbone: require('backbone'),
+    loglevel: logging,
+    _: require('lodash')
+};
+
+var Kolibri = _.extend({
+    lib: lib,
+    plugins: Mediator._plugin_registry
+}, _.pick(Mediator, public_methods));
```

Comment:
`_.pick` takes certain attributes (the keys in `public_methods`) from the Mediator object and then `_.extend` adds them to the object that has the `lib` and `plugins` attributes defined.